src/server/services/procedures/phone-iot/common.js

```diff
@@ -60,4 +63,26 @@ utils.gracefulPasswordParse = function(password) {
     return res;
 }
 
+// given some image source, attempts to convert it into a buffer for sending over UDP.
+// if the image type is not recognized, throws an error.
+utils.prepImageToSend = async function(raw) {
+    let matches = raw.match(/^\s*\<costume .*image="data:image\/png;base64,([^"]+)".*\/\>\s*$/);
+    if (matches) {
+        const raw = Buffer.from(matches[1], 'base64');
+        const img = await sharp(raw)
```

Comment:
Could you use `jimp` instead? (That is the image manipulation library already used in NetsBlox.) It's not a deal breaker but would be nice to use a single image manipulation dependency instead of multiple, if possible.

Reply:
Oh, sure, as long as it supports read/write of png/jpeg binary that's all I really need. I'll see if I can switch that over.

Reply:
Ok, I switched that over to `jimp` and removed the `sharp` dependency.